test/testsuites/packagestests.go

```diff
@@ -102,7 +112,116 @@ func testOpSrcWithConflictingPackages(t *testing.T) {
 	err = helpers.CreateRuntimeObject(test.Global.Client, ctx, helpers.CreateOperatorSourceDefinition(opSrcName, namespace))
 	assert.NoError(t, err, "Could not create operator source")
 
+	// Check that the Deployment was created.
+	resultDeployment := &apps.Deployment{}
+	err = helpers.WaitForResult(client, resultDeployment, namespace, opSrcName)
+	assert.NoError(t, err)
+
+	// Now check that the Deployment is ready.
```

Comment:
Why are you checking for just the deployment? Shoudln't you ensure all child resources are created.

Reply:
That's a good idea - I had only added this line to make sure that the CSC was not rolled out before the `OpSrc` with conflicting packages was created.

Reply:
I really don't understand what you mean by `OperatorSource` with conflicting packages. How can an `OperatorSource` contain conflicting packages?

Reply:
>OK, that description is hard to parse. Is it saying that I can add two OperatorSources that have some identical packages?

Identical packages is a better way to describe the situation.